### ISSUE TITLE
Homework #2 -- Relaxr

### ISSUE THREAD

**Issue by celestelayne** [nicholasashby/FEWD40HOMEWORK#1]
# FEWD Week #2: Styling with CSS

---
### Description

A hot, new SF Tech Startup, Relaxr, has approached you to help them develop a new landing page for their company. They've handed you the design file for the site, along with the copy and assets. You need to take these files and turn it into a landing page.

Student: Nick Ashby
Project URL: https://github.com/nicholasashby/FEWD40HOMEWORK/tree/master/Homework/Relaxr

| Technical Requirements | Does Not Meet Expectations (R) | Meets Expectations (Y) | Exceeds Expectations (G) |
| --- | --- | --- | --- |
| Use HTML5 structural elements (nav, header, footer) |  | X |  |
| Use a CSS Reset file in addition to your style.css file to style the page |  |  | X |
| Use IDs and Classes to to select and style elements on the page |  |  | X |
| Style your text with the Google Fonts provided by your style guide |  |  | X |
| Follow naming conventions, maintain consistency across .html and .css files and use best practices for naming IDs and Classes |  |  | X |
| Indent nested elements to increase your code's readability |  |  | X |
| TOTAL: Exceeds Expectations |  |  |  |

**Comment by nicholasashby** [comment]
Hi - was this my report? The name says Rose Yuan.

Thanks!

Nick

**Comment by celestelayne** [comment]
Yes...look in GitHub and you'll see yours. I did hers first and copied but
accidentally saved yours before I added your name.

Celeste

On Sat, Feb 6, 2016 at 2:35 PM, Nick Ashby notifications@github.com wrote:

> Hi - was this my report? The name says Rose Yuan.
> 
> Thanks!
> 
> Nick
> 
> On Fri, Feb 5, 2016 at 9:47 PM, Celeste notifications@github.com wrote:
> 
> > ## #FEWD Week #2: Styling with CSS
> > 
> > ###Description
> > 
> > A hot, new SF Tech Startup, Relaxr, has approached you to help them
> > develop a new landing page for their company. They've handed you the
> > design
> > file for the site, along with the copy and assets. You need to take these
> > files and turn it into a landing page.
> > 
> > Student: Rose Yuan
> > Project URL: https://github.com/rjyuan/GA-FEWD/tree/master/relaxr
> > Technical Requirements Does Not Meet Expectations (R) Meets Expectations
> > (Y) Exceeds Expectations (G)
> > Use HTML5 structural elements (nav, header, footer) X
> > Use a CSS Reset file in addition to your style.css file to style the page
> > X
> > Use IDs and Classes to to select and style elements on the page X
> > Style your text with the Google Fonts provided by your style guide X
> > Follow naming conventions, maintain consistency across .html and .css
> > files and use best practices for naming IDs and Classes X
> > Indent nested elements to increase your code's readability X
> > TOTAL: Exceeds Expectations
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/nicholasashby/FEWD40HOMEWORK/issues/1.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nicholasashby/FEWD40HOMEWORK/issues/1#issuecomment-180875573
> .

## 

celeste layne
san francisco :: ca
